.github/workflows/ci.yml

```diff
@@ -7,15 +7,17 @@ on:
     tags: [v*]
 jobs:
   ci:
-    # run on external PRs, but not on internal PRs since those will be run by push to branch
+    # Run on external PRs, but not on internal PRs since those will be run by push to branch
+    # For JVM 3 GB is the lowest value we can set without causing failures in tests
+    # For Native we probably can lower the value further
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
-        target-platform: [ "JVM", "JS", "Native" ]
+        target-platform: [ "JVM", "Native" ]
     env:
-      JAVA_OPTS: "-Xmx3500M -Dsbt.task.timings=true" # -Xlog:gc -XX:+PrintGCDetails -Xlog:gc*::time
+      JAVA_OPTS: "-Xmx3000M -Dsbt.task.timings=true" # -Xlog:gc -XX:+PrintGCDetails -Xlog:gc*::time
```

Comment:
maybe we can have a single job, with conditional environment variables? See https://dev.to/ku6ryo/how-to-set-env-parameters-conditionally-in-github-actions-1g6a